File: android/DFM/src/main/java/com/dontflymoney/userdata/SP.java
package com.dontflymoney.userdata;

import android.content.Context;
import android.content.SharedPreferences;

class SP
{
	private static final String sharedPreferencesDFM = "DfM";

	public static String getValue(Context context, String key)
	{
		SharedPreferences sp = context.getSharedPreferences(sharedPreferencesDFM, Context.MODE_PRIVATE);
		return sp.getString(key, null);
	}

	public static void setValue(Context context, String key, String value)
	{
		SharedPreferences sp = context.getSharedPreferences(sharedPreferencesDFM, Context.MODE_PRIVATE);
		SharedPreferences.Editor edit = sp.edit();

		edit.putString(key, value);
		edit.apply();
	}
}

File: android/DFM/src/main/kotlin/com/darakeon/dfm/user/SP.kt
package com.darakeon.dfm.user

import android.content.Context

internal object SP {
	private val sharedPreferencesDFM = "DfM"

	fun getValue(context: Context, key: String): String {
		val sp = context.getSharedPreferences(sharedPreferencesDFM, Context.MODE_PRIVATE)
		return sp.getString(key, "")
	}

	fun setValue(context: Context, key: String, value: String) {
		val sp = context.getSharedPreferences(sharedPreferencesDFM, Context.MODE_PRIVATE)
		val edit = sp.edit()

		edit.putString(key, value)
		edit.apply()
	}
}
